lbuild/format.py

```diff
@@ -134,11 +139,11 @@ def __str__(self):
         return string + ansi_escape("reset")
 
 
-c = color_str
+_cw = ColorWrapper
```

Comment:
This `ColorWrapper` stuff is super hacky, but I it's worth the pain to have the output formatted.